pages/detail/book/[id]/detail.js

```diff
@@ -19,7 +19,7 @@ export default function DetailBookPage({ data }) {
       <StyledCard>
         <StyledId>
           <span>ID:</span>
-          <span>{currentBook.id}</span>
+          <span>{currentBook.internalId}</span>
```

Comment:
`internalId` is not an ideal name, I also saw this in your data.
If your `internalId` is just to display the number of the book in the list, how about calling this key `number`?